lms/models/lti_params.py

```diff
@@ -138,8 +138,28 @@ def _to_lti_v11(v13_params):
         v11_params.update({f"custom_{key}": value for key, value in custom.items()})
 
     if "roles" in v11_params:
-        # We need to squish together the roles for v1.1
-        v11_params["roles"] = ",".join(v11_params["roles"])
+        context_roles = []
+        for role in v11_params["roles"]:
+            # From: https://www.imsglobal.org/spec/lti/v1p3#role-vocabularies
+            #
+            # Conforming implementations MAY recognize the simple
+            # names for context roles; thus, for example, vendors can use
+            # the following roles interchangeably:
+            #   http://purl.imsglobal.org/vocab/lis/v2/membership#Instructor
+            #   Instructor
+            if "http://purl.imsglobal.org/vocab/lis/v2" not in role:
+                # If using the simple name, we'll take it
+                context_roles.append(role)
+
+            if role.startswith("http://purl.imsglobal.org/vocab/lis/v2/membership"):
+                # For roles that have the whole LIS 2.0 name, take only the one relevant for the current context.
+                # We'd need to expose the rest of the roles somewhere else if
```

Comment:
How we do this, exposing the other type of riles, is to be discussed.

We could add extra fields on lti_user for example but I have the impression that when we need to know "is this user a teacher anywhere in the institution" we could reach directly into the LTI1.3 while keeping the rest of the roles notion on the app as the more straightforward "role in the current launch/course".

Another alternative could be to change this at the `is_learner` `is_teacher` level but from a bug fix point of view I reckon that more complex and more error prone (we only do `"instructor" in roles.lower()` now.

Not sure what the implications of this change are for the LTIRole model. I reckon the mapping of any of the institutions roles to eg just "instructor" are wrong, when looking at the DB and seeing instructor we have a clear understanding of what that means, we shouldn't have to check the original LTI string to filter further. 

We could do a DB migration cleanup after this if necessary.